packages/camera/camera_avfoundation/ios/Classes/FLTCam_Test.h

```diff
@@ -5,6 +5,37 @@
 #import "FLTCam.h"
 #import "FLTSavePhotoDelegate.h"
 
+/**
+ A block type definition named VideoDimensionsForFormatBlock.
```

Comment:
See previous comments; this is a restatement of the line of code, and thus doesn't add any information.